tests/transforms/test_mitigate.py

```diff
@@ -14,7 +14,7 @@
 """
 Tests for mitigation transforms.
 """
-# pylint:disable=no-self-use
+# pylint:disable=no-self-use, unnecessary-lambda-assignment
```

Comment:
Is there unnecessary-lambda-assignment added in this PR?

Reply:
Nope, it was there already. The `pylint` is run on older test files only when they are updated, so I guess that's why this wasn't added before.